src/Workspaces/CSharp/Portable/Extensions/ContextQuery/CSharpSyntaxContext.cs

```diff
@@ -195,6 +198,10 @@ private static CSharpSyntaxContext CreateContextWorker(Workspace workspace, Sema
                                             targetToken.Parent.IsKind(SyntaxKind.DestructorDeclaration) &&
                                             targetToken.Parent.Parent.IsKind(SyntaxKind.ClassDeclaration, SyntaxKind.StructDeclaration);
 
+            var isFirstDotOfDotDotToken = targetToken.IsKind(SyntaxKind.DotDotToken) &&
+                                                targetToken.Parent.IsKind(SyntaxKind.RangeExpression) &&
+                                                position == targetToken.SpanStart + 1;
```

Comment:
1. 'isAfterFirstDotOfDotDot` would probably be a better name.
2. i don't think we want to check for the parent.  We'd likely want this behavior anywhere since we're basically just saying "is hte user after a dot?"